src/status_im/ui/screens/contacts/contact_list/views.cljs

```diff
@@ -11,6 +11,8 @@
             [status-im.ui.screens.contacts.views :as contact-options]
             [status-im.i18n :as i18n]))
 
+;; TODO(alwx): this namespace is not used; should it be removed?
```

Comment:
If it is not used, please remove yes!

Reply:
@jeluard but what's the future of contacts list in general? :) maybe it's not wise to remove it because it will be used soon